aeron/clientconductor.go

```diff
@@ -56,6 +56,8 @@ const (
 	heartheatRegistrationIdOffset = int32(0)
 )
 
+var TemporaryError = errors.New("temporary error")
```

Comment:
What is the significance of the TemporaryError?

Reply:
I see it now.  Maybe define this closer to 
```golang
func (cc *ClientConductor) FindSubscription(registrationID int64) (*Subscription, error) {
```

Reply:
Yes, exactly.  I initially had it towards the top since I view it as const-like, even though you're not allowed to initialize constants with errors.New().  But thanks for shining light on this part of the code - I realized the text of the error didn't match the functionality.  Fixed the text in latest commit.